api/tests/phones_views_tests.py

```diff
@@ -48,6 +55,16 @@ def mocked_twilio_validator():
         yield mock_twilio_validator
```

Comment:
Nit: Add a default `validate` returning `True` in the fixture?

```python
mocked_twilio_validator.validate = Mock(return_value=True)
```

You could then delete that line from tests using `mocked_twilio_validator`. For the tests returning `False`, you'd need:

```python
mocked_twilio_validator.validate.return_value = False
```